### DISCUSSION TITLE
How to get containers (qBittorrent, sabnzbd, jackett, sonarr, radarr, bazarr) going through a NordLynx (NordVPN + Wireshark) VPN container Without unRAID  (docker native or swarm mode)

### DISCUSSION THREAD

**Discussion by TheLazyGeekGuy** [bubuntux/nordlynx#62] [Q&A]
How to get containers (qBittorrent, sabnzbd, jackett, sonarr, radarr, bazarr) going through a NordLynx (NordVPN + Wireshark) VPN container Without unRAID in docker native or swarm mode ?

**Comment by yash-fn** [comment]
Can't you just force those containers to use nordlynx's network stack like posited in the README? Never used unraid or swarm modes, so not sure if applicable.
